.env.example

```
@@ -23,6 +30,9 @@ LOG_MAX_AGE_DAYS=30
 # セッションログの最大合計サイズ（MB、デフォルト: 100MB）
 LOG_MAX_SIZE_MB=100
 
+# データ永続化
+DATA_DIR=                  # データ保存ディレクトリ（デフォルト: ワークスペース/.pocket-dev-relay）
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)